### DISCUSSION TITLE
Router Firmware

### DISCUSSION THREAD

**Discussion by mhamzahkhan** [darkxst/silabs-firmware-builder#85] [General]
Hello everyone,

I'm not sure if this is the best place to ask, but I have two SONOFF ZBDongle-E dongles. One is my coordinator, and one is flashed with the [router firmware](https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E/Router), which I'm using to extend the range of my Zigbee network. (I know it's overkill, but I don't have any wired Zigbee devices on my network that function as router.)

On my coordinator dongle, I'm running 7.4.1.0 from this repository, and it's working great, but I was wondering if there is any newer router firmware by any chance? :)

**Comment by tl-sl** [comment]
The 6.10 router firmware build from iTead works fine, no need for an updated version here for ZBDongle-E. Stick with that for now.

I will be adding router firmwares here later since most of the other devices in this repo can't benefit from iTead builds.

**Reply by Hedda** [reply]
@tl-sl please see/join diacussion here -> https://github.com/darkxst/silabs-firmware-builder/issues/15

**Reply by darkxst** [reply]
That is just me, I replied with the wrong account!

**Comment by Hedda** [comment]
> On my coordinator dongle, I'm running 7.4.1.0 from this repository, and it's working great, but I was wondering if there is any newer router firmware by any chance? :)

See this existing feature request -> https://github.com/darkxst/silabs-firmware-builder/issues/15

> One is my coordinator, and one is flashed with the [router firmware](https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E/Router), which I'm using to extend the range of my Zigbee network. (I know it's overkill, but I don't have any wired Zigbee devices on my network that function as router.)

That is not overkill at all, you should really have many more Zigbee Router devices than that, see -> https://community.home-assistant.io/t/zigbee-networks-how-to-guide-for-avoiding-interference-optimizing-using-zigbee-router-devices-repeaters-extenders-to-get-best-possible-range-and-coverage/515752
